packages/trader/src/AppV2/Components/TradeParameters/Strike/strike.tsx

```diff
@@ -54,7 +54,25 @@ const Strike = observer(({ is_minimized }: TStrikeProps) => {
         },
         {
             id: 2,
-            component: <TradeParamDefinition description={<Localize i18n_default_text='Content goes here.' />} />,
+            component: (
+                <TradeParamDefinition
+                    description={
+                        <>
+                            <p>
+                                <Localize i18n_default_text='It is the price where you can start receiving a payout from an option.' />
+                            </p>
+                            <br />
```

Comment:
May we avoid using `<br/>`?  You can always add a className to `<p>`

Reply:
why?

Reply:
We are adding styles not by using tags, but by applying classes + `<br/>` is not semantic (only if you're writing poetry).
Not critical, but up to you